TestDockerfileServer

```diff
@@ -0,0 +1,13 @@
+FROM golang:alpine
```

Comment:
Like; ideally the container you use for integration testing the server is the same as what would be used for an actual deployment.